app/services/concerns/xccdf/rules.rb

```diff
@@ -8,7 +8,9 @@ module Rules
     included do
       def save_rules
         @rules ||= @op_rules.map do |op_rule|
-          ::Rule.from_openscap_parser(op_rule, benchmark_id: @benchmark&.id)
+          ::Rule.from_openscap_parser(op_rule,
+                                      benchmark_id: @benchmark&.id,
+                                      group_id: Group.find_by(ref_id: op_rule.parent_id)&.id)
```

Comment:
@skateman 
We talked about adding a `parent_id` attribute onto Rule to set the parent id but I thought about it a little and I was thinking that we could have Rule belong to Group and set group_id like this. So `group_id` will be nil if the parent is `Benchmark` but it'll be set if the parent is Group. We also have the `parent_type` attribute on Rule to check what the parent type is.
Using `Group.find_by` might slow things down though.. I could add an if statement to only pass the `group_id` parameter if the `parent_type == 'Group'` or I could think of a better way to find the group id 😄